jenkins-ci/process.py

```diff
@@ -0,0 +1,64 @@
+
+import sys
+import os
+import argparse
+import lib.bisect_lib as cb
+# from lib import common_lib as cb
+def fetch(sid,type):
+    # print("1",path,"1")
+    rrjson={}
+    rjson=cb.read_json(cb.base_path+sid+'/'+sid+'.json')
+    # print(rjson)
```

Comment:
If not required remove these print statements.

Reply:
okay